Scripts/RemediationScripts/Remediate-ConfigureAADAuthExtOnLinuxVM.ps1

```diff
@@ -0,0 +1,1059 @@
+<###
+# Overview:
```

Comment:
Can you provide screenshots of various scenarios of the script running?

Reply:
OneNote has been shared.